skrifa/src/variation.rs

```diff
@@ -333,9 +334,8 @@ impl<'a> NamedInstance<'a> {
     /// Computes a location in normalized variation space for this instance and
     /// stores the result in the given slice.
     ///
-    /// # Panics
-    ///
-    /// Panics if `location.len() < self.user_coords().count()`.
+    /// If `location.len() < self.user_coords().count()`, the omitted
+    /// coordinates will assume their default positions.
```

Comment:
I like the If not setting is provided... language over the len comparison for reasons noted in the comment above.

Reply:
Just deleted this as per suggestion above. In this case, all settings should be provided because they come from the named instance.